lib/config/rules/typescript.js

```diff
@@ -0,0 +1,41 @@
+// Typescript Specific:
+// These rules find errors related to TypeScript features
+// Reference https://palantir.github.io/tslint/rules/
+
+module.exports = {
+  // Enforces function overloads to be consecutive.
+  'adjacent-overload-signatures': 'off',
```

Comment:
Forgot to update the typescript.js file to use `tslint`'s `true` / `false` rule convention instead of the `off` / `warn`/  `error` like `eslint`